### DISCUSSION TITLE
治安局招新啦！

### DISCUSSION THREAD

**Discussion by yjverse6** [share121/inter-knot#5293] [绝区零]
![微信图片_20240922131731](https://github.com/user-attachments/assets/a558db6a-03e5-4c2b-a5cf-fa600d2284ff)
![微信图片_20240922131720](https://github.com/user-attachments/assets/4b796dea-534e-4859-b32a-42fa22550e31)
![微信图片_20240922131740](https://github.com/user-attachments/assets/330e66a9-c925-4dcc-90e0-c98ccaa0bcdc)

**Comment by yansai0v0** [comment]
wok治安官😰

**Comment by RGthx** [comment]
谁家治安官绳网招新啊？招安绳匠免培训吗？

**Comment by IXfromThorne** [comment]
“我抓我自己？”这句话已经说腻了（嗯呢，嗯呢呢）

**Comment by TNCNET** [comment]
虽然但是，我得问问贵局在这招募是怎么招募？
